src/eval.rs

```diff
@@ -301,10 +301,12 @@ pub fn create_ecx<'mir, 'tcx: 'mir>(
             )?;
         }
         EntryFnType::Start => {
+            let sigpipe = 2; // Inlining of `DEFAULT` from https://github.com/rust-lang/rust/blob/master/compiler/rustc_session/src/config/sigpipe.rs
+            let sigpipe_arg = Scalar::from_u8(sigpipe);
             ecx.call_function(
```

Comment:
There's another `call_function` a few lines up, that one probably also needs to be adjusted.

Reply:
Oops, yes, I got it backwards in my head, I should change that place *instead* of this place.

All tests pass locally for me now, hopefully it all works in CI too.